be/src/formats/orc/apache-orc/c++/include/orc/LoadType.h

```diff
@@ -0,0 +1,14 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2022-present, StarRocks Inc.
+
+#pragma once
+
+namespace orc {
+
+enum class LoadType {
+    NO_LAZY_LOAD = 0,
```

Comment:
I don't understand why there is `no_lazy_load` in it.  My intutiion is we only need `bool` to represent it(2 states). Could you make confirmation we have to use 3 states?

Reply:
{
  a: int,
  b: {
    c: int,
    d: double
  }
}

For example, I want to set `c` as an active field, `d` as a lazy load field.
If I set `b` as an active field, `d` will not be used when we do `loadLazyLoadNext()`.
If I set `b` as a lazyload field, `c` will not be used when we do `next()`.

So we need the third state for this situation.

Reply:
fields can be marked as lazy or not indepdently.

If c is active field, d is not. then b should be active field. For any compound type, if any subfield is acitve field, then this compound type is active.